core/src/test/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender42Test.java

```diff
@@ -0,0 +1,16 @@
+package com.microsoft.applicationinsights.internal.channel.common;
```

Comment:
Actually I think you can merge this file and the next file (`ApacheSender43Test.java`) into a single file named `ApacheSenderTest.java` and just have different test names instead.